Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/* /var/lib/apt/archives/*.deb
 COPY Gemfile Gemfile.lock /app/
 RUN bundle install --jobs 4 \
-  && bundle binstubs --all --path /usr/local/bundle/bin
+  && bundle binstubs --all --path /usr/local/bundle/bin && bundle binstubs bundler --force
```

Comment:
I think it might be clearer if this new command was split onto a new line.

Reply:
Fixed in 6e3b5d7